index.bs

```diff
@@ -8520,12 +8520,18 @@ is the [=set/intersection=] of the relevant construct's [=own exposure set=]
 with the the [=host interface=]'s [=exposure set=].
 Otherwise, it is the [=host interface=]'s [=exposure set=].
 
-An [=interface=], [=namespace=], [=interface member=], or [=namespace member=] is
-<dfn id="dfn-exposed" export>exposed</dfn> in a given ECMAScript global environment if the
-ECMAScript global object implements an interface that is in the construct's [=exposure set=], and either:
-
-*   the [=relevant settings object=] for the ECMAScript global object is a [=secure context=]; or
-*   the construct is not [=available only in secure contexts=].
+<div algorithm>
+    An [=interface=], [=namespace=], or [=member=] |construct| is <dfn id="dfn-exposed" export>exposed</dfn>
+    in a given ECMAScript global environment |env| if the following steps return true:
+    
+    1.  If |env| does not implement an [=interface=] that is in |construct|'s [=exposure set=],
```

Comment:
|env| does not implement interfaces; |env|'s `[=realm/global object=]` does. This got lost in translation.

Reply:
Thanks.